app/components/dashboard/collection_component.rb

```diff
@@ -27,8 +25,8 @@ def deposits
     end
 
     sig { returns(T::Boolean) }
-    def deposit_blocked?
-      NO_DEPOSIT_STATUS.include? collection.state
+    def deposit_allowed?
```

Comment:
Rather than doing this, can we just modify the query in WorkPolicy so it excludes these collections from appearing?

Reply:
Looking. Thanks for the suggestion.